app/models/order.rb

```diff
@@ -13,4 +13,13 @@ def grandtotal
   def quantity
     items.count
   end
+
+  def cancel_order
+    item_orders.where(status: 'fulfilled').each do |order|
+      order.item.update(inventory: (order.quantity + order.item.inventory))
+    end
+
```

Comment:
need to ask about this one in the checkin on monday

Reply:
Just thinking to myself if should this be an order model method or an item  model method?